modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc

```diff
@@ -6,6 +6,8 @@
 
 This topic describes how to use the Redpanda Helm chart to configure and deploy Redpanda Connectors in Kubernetes.
 
+include::shared:partial$community-supported-connectors.adoc[]
+
 Redpanda Connectors is a package that includes glossterm:Kafka Connect['Kafka Connect is a framework that provides a consistent and reliable platform to run and manage connectors.'] and the following built-in glossterm:connectors['Connectors are the plugins that interface with external systems. Connectors are plugged into Kafka Connect to facilitate data flow between Redpanda and these external systems.'], sometimes known as plugins. You can create and configure the built-in connectors in Redpanda Console.
```

Comment:
```suggestion
Redpanda Connectors is a package that includes Kafka Connect and the following built-in connectors, sometimes known as plugins. You can create and configure the built-in connectors in Redpanda Console.
```

We don't need these definitions now that the partial provides a link to info about Kafka Connect.